contrib/mod_tls.c

```diff
@@ -10332,29 +10325,6 @@ static int tls_verify_cb(int ok, X509_STORE_CTX *ctx) {
 
       if (strcasecmp(mech, "crl") == 0) {
         ok = tls_verify_crl(ok, ctx);
-        if (!ok) {
-          int crl_verify_err = 0;
-
-#if OPENSSL_VERSION_NUMBER >= 0x10100000L
-          crl_verify_err = X509_STORE_CTX_get_error(ctx);
-#else
-          crl_verify_err = ctx->error;
-#endif /* OpenSSL-1.1.x and later */
-
-         /* If we use the wrong public key to verify the CRL (as for an empty
-          * CRL, or a CRL signed by a different CA/key than the offered cert),
-          * then this could fail in an expected manner (Bug #4468).
-          */
-          if (crl_verify_err == X509_V_ERR_CRL_SIGNATURE_FAILURE) {
```

Comment:
This `X509_V_ERR_CRL_SIGNATURE_FAILURE` would occur because of the overly-complex previous implementation, where we could (and would) use the wrong public key to verify the signature on a CRL.  Since that implementation has been redone, this particular error/code path should not occur going forward.